spec/gogo/x/gov/codec.proto

```diff
@@ -123,6 +123,18 @@ message ProposalCommon {
   }
   // Result is the final result based on the votes and election rule. Initial value is Undefined.
   Result result = 12;
+  enum ExecutorResult {
+    // An empty value is not allowed
+    PROPOSAL_COMMON_EXECUTOR_RESULT_INVALID = 0 [(gogoproto.enumvalue_customname) = "ExEmpty"];
```

Comment:
I think it might be better to keep the Go name `Invalid`. With current `ExEmpty` one may think that this is a value that is stored before there is a result. Empty being the opposite of non-empty - a result. This is not true as zero is not a valid value and `NotRun` is the "empty" value.

Reply:
The problem is that these enums are class global:

There is 
ProposalCommon_Invalid for Status
ProposalCommon_Empty for Result and
ProposalCommon_ExEmpty for ExecutorResult

I had named this Invalid at first, but then got compilation errors due to naming clashes.
I am happy to name them _InvaliStatus, _InvalidResult, _InvalidExecution.
The default names would be PROPOSAL_COMMON_EXECUTOR_RESULT_INVALID but I believe that was not wished

Let me know another set of names (no clashes), and I am happy to adjust

Reply:
How about follow the trend and call it [`RealInvalid`](https://twitter.com/realdonaldtrump)? :joy_cat: 

Makes sense. How about `InvalidState` with a comment that `Invalid` is already taken in this namespace?

Reply:
Yeah, go enums generated from protobuf definitions have a habit of bein global, so they end up containing namespace and being messy.

Reply:
Giving it more thought, do we care for the name at all. Why not removing `gogoproto.enumvalue_customname` declaration for this value as it should never be used anyway?

Reply:
Removed customnames from these. Pushed and waiting for CI :crossed_fingers: